.git-blame-ignore-revs

```diff
@@ -0,0 +1,8 @@
+# style: end-of-file-fixer pre-commit hook
+b372d66ab9e9aa30573f240d87a6ea92474cce75
+
+# style: trailing-whitespace pre-commit hook
+0f0a977d8aaba773570dff8f828c358df22203aa
+
+# style: check-docstring-first pre-commit hook
+1a36dc83af39159749ce5857eca6ddacd345e6bb
```

Comment:
Note to reviewer: As this file contains commits it is sensitive to rebases, and so please make sure that these commits are correct if anyone rebases this during review.

Reply:
This is great that this is possible.  Really helpful for big cosmetic changes like this.

Reply:
I rebased, but I also updated the commits.

```console
$ git log --graph --oneline --decorate --all | head -n 10
* 02fc0611cf (HEAD -> mnt/apply-pre-commit-to-all-files, origin/mnt/apply-pre-commit-to-all-files) DOC: Add pre-commit hook run dev instructions
* f3aa826d83 DEV: Add .git-blame-ignore-revs to ignore style in blame view
* 66c12f2778 STY: Apply check-docstring-first pre-commit hook
* 33516082c5 STY: Apply trailing-whitespace pre-commit hook
* 94d985a744 STY: Apply end-of-file-fixer pre-commit hook
* 7e04925fcd MNT: Update pre-commit hooks to latest versions
| * 3dd6049160 (mnt/add-codespell-to-pre-commit-hooks) FIX: Fix typo caught by codespell pre-commit hook
| * 19987645ec DEV: Add codespell to pre-commit hooks
|/  
*   19992285cb (upstream/main, origin/main, main) Merge pull request #22807 from anntzer/qd
```